build/automation/var/profile/dev.mk

```diff
@@ -88,16 +88,11 @@ COGNITO_USER_POOL_CLIENT_ID := $(or $(COGNITO_USER_POOL_CLIENT_ID), )
 COGNITO_USER_POOL_ID := $(or $(COGNITO_USER_POOL_ID), )
 ADD_DEFAULT_COGNITO_USERS := false
 
-POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-pca-dev-uec-dos-api-pc-ingress.k8s-nonprod.texasplatform.uk/api
+POSTCODE_MAPPING_SERVICE_URL := http://mockservice.sfs.test:8080/api
```

Comment:
Is it possible to rename this URL so it's clearer what it is. Something like: mock.postcodeapi.sfs:8080/api ?